lib/block_view_types.go

```diff
@@ -919,6 +937,28 @@ type UtxoOperation struct {
 	// PrevLockedStakeEntries is a slice of LockedStakeEntries
 	// prior to a unstake or unlock stake txn.
 	PrevLockedStakeEntries []*LockedStakeEntry
+
+	//
+	// Coin Lockup fields
+	//
+
+	// PrevLockedBalanceEntry is the previous LockedBalanceEntry prior
+	// to a DAO coin lockup.
+	PrevLockedBalanceEntry *LockedBalanceEntry
```

Comment:
No action necessary here, but Proof of Stake should make disconnects no longer valid. If Sofonias is at a point where this is the case, it's possible we can make this the first txn that doesn't need disconnects, and therefore that doesn't need all this prev crap.

Otherwise, it will be the last txn we'll ever write that has this...

Reply:
Piotr brought this up as well about a week ago. Are there potential benefits to keeping disconnects around? I was thinking possible for debugging, testing, or emergency reversals. Not sure if any of those are worthwhile concerns.